### PR TITLE
feat(Tabs): Implement overflow styles for secondary tabs

### LINKED ISSUE #1705: Implement overflow styles for secondary tabs
https://github.com/patternfly/patternfly-next/pull/1606

@jschuler for PR review

### PR DESCRIPTION
**Pull request by jessiehuff**
Fixes #1705

### CODE REVIEW

File: packages/patternfly-4/react-core/src/components/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -239,15 +237,13 @@ export class Tabs extends React.Component<TabsProps, TabsState> {
               </li>
             ))}
           </ul>
-          {!isSecondary && (
             <button
```
Comment (jschuler) [inline packages/patternfly-4/react-core/src/components/Tabs/Tabs.tsx]: The buttons should also get the modifier `pf-m-secondary` for secondary tabs

Comment (jessiehuff) [reply packages/patternfly-4/react-core/src/components/Tabs/Tabs.tsx]: Ok, thanks! :)

File: packages/patternfly-4/react-core/src/components/Tabs/Tabs.tsx [hunk 2]
```diff
@@ -239,15 +238,13 @@ export class Tabs extends React.Component<TabsProps, TabsState> {
...
-              className={css(styles.tabsScrollButton)}
+              className={css(styles.tabsScrollButton, isSecondary && buttonStyles.modifiers.secondary)}
               aria-label={rightScrollAriaLabel}
```
Comment (mcoker) [inline packages/patternfly-4/react-core/src/components/Tabs/Tabs.tsx]: This reads "Scroll Right" but should be "Scroll right" (sentence case), and match `leftScrollAriaLabel`.

Comment (jessiehuff) [reply packages/patternfly-4/react-core/src/components/Tabs/Tabs.tsx]: Ok, nice catch! :)

**Pull request merged**